src/validation_workflow/docker/cleanup_docker.py

```diff
@@ -0,0 +1,34 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import subprocess
+from subprocess import PIPE
+
+yml_template = 'docker-compose.yml'
+
+"""
+This class is used to stop&remove docker containers created at step 3, and remove docker-compose.yml in /tmp/.
+"""
+
+
+class CleanupDocker:
```

Comment:
I dont think even you need a separate class just to clean up. It can be a method under docker  class as well.

Reply:
oh.. sure .. I can certainly add cleanup into other class as a method. I was trying to seperate each function out to be an individual class. Maybe this is not a good practice to do ,. LOL